tests/governance.py

```diff
@@ -472,9 +472,6 @@ def add(parser):
         governance_history.run,
         package="samples/apps/logging/liblogging",
         nodes=infra.e2e_args.max_nodes(cr.args, f=0),
-        # Higher snapshot interval as snapshots trigger new ledger chunks, which
-        # may result in latest chunk being partially written
-        snapshot_tx_interval=10000,
```

Comment:
I believe this was set because of old limitations of the Python infra which didn't cope well with ledger chunks that were in progress.